typescript/cli/src/commands/validator.ts

```diff
@@ -49,3 +64,60 @@ const addressCommand: CommandModuleWithContext<{
     process.exit(0);
   },
 };
+
+const preFlightCheckCommand: CommandModuleWithContext<{
+  chain: string;
+  validators: string;
+}> = {
+  command: 'preflightCheck',
```

Comment:
```suggestion
  command: 'preflight-check',
```
Seems the pattern so far has been `foo-bar` for multi word commands in the CLI